src/pages/index.js

```diff
@@ -274,20 +273,20 @@ avatarImageFormValidator.enableValidation();
 
   cardSection.renderItems();
 });
- */
+
 /* Fetch and displaying User name, job bio, and profile picture */
-/* api.getUserInformation().then((userData) => {
+api.getUserInformation().then((userData) => {
```

Comment:
You need to combine the requests of the user's info and the cards together in `Promise.all`